src/app/code/local/Aligent/GeoIP/Helper/Data.php

```diff
@@ -197,12 +197,17 @@ private function geoipOpen($filename, $flags) {
                 $datFilePath = $dir . DS . $filename;
                 if (strpos($datFilePath, '/') !== 0) {
                     // First look for file on the include path
-                    if ((function_exists('stream_resolve_include_path') && $_filename = stream_resolve_include_path($datFilePath)) === false) {
-                        // Then look for the file relative to the Magento root.
-                        if (!file_exists($_filename = Mage::getBaseDir() . DS . $datFilePath)) {
-                            throw new Exception (sprintf('Unable to find file: "%s"', $datFilePath));
```

Comment:
@willtran 
:disappointed: looks like the geoip function actually has a `or die` statement in it instead of exception throw. I'll add in an exception if the file isn't found, thanks.